pkg/discovery/monitoring/kubelet/kubelet_monitor.go

```diff
@@ -466,11 +493,16 @@ func (m *KubeletMonitor) parseContainerStats(pod *stats.PodStats, timestamp int6
 	return totalUsedCPU, totalUsedMem, allMetricsMissing
 }
 
-func (m *KubeletMonitor) genThrottlingMetrics(etype metrics.DiscoveredEntityType, key string, throttled, total float64, timestamp int64) {
+func (m *KubeletMonitor) genThrottlingMetrics(etype metrics.DiscoveredEntityType, key string, tm *throttlingMetric, timestamp int64) {
+	cpuLimits := float64(0)
+	if tm.cpuQuota != 0 && tm.cpuPeriod != 0 {
+		cpuLimits = tm.cpuQuota * 1000 / tm.cpuPeriod
+	}
```

Comment:
To test this logic, should we add some test cases to cover the zero limit scenarios?

Reply:
Good point. Will add additional unit tests for zero limit